src/main/java/io/fabric8/forge/generator/pipeline/ChoosePipelineStep.java

```diff
@@ -217,6 +216,18 @@ public String convert(PipelineDTO pipeline) {
         LOG.debug("initializeUI took " + watch.taken());
     }
 
+    private boolean isImportRepositoryFlow(Map<Object, Object> attributeMap) {
+        Object object = attributeMap.get(BoosterDTO.class); // Booster is a step for Quickstart flow only
+        if (object == null) {
```

Comment:
can be one liner `return attributeMap.containsKey(BoosterDTO.class)`

Reply:
oki, let me revisit that one.